frontend/src/common/components/Organism/Header.vue

```diff
@@ -28,8 +28,7 @@ const modalContent = ref<'login' | 'signup' | 'password' | 'close'>('close')
       <div class="flex w-full max-w-7xl items-center justify-between gap-5">
         <div class="flex w-1/2 min-w-fit items-center justify-between gap-8">
           <RouterLink to="/" class="flex items-center gap-1">
-            <img :src="Logo" alt="logo" width="45" />
-            <h1 class="text-2xl font-bold">Codedang</h1>
+            <img :src="Logo" alt="logo" height="1" />
```

Comment:
```suggestion
            <img :src="Logo" alt="logo" width="90"/>
```

대충 90정도로 주면 보기 좋은 것 같아요!

Reply:
<img width="1575" alt="Screenshot 2023-07-30 at 10 10 15 PM" src="https://github.com/skkuding/codedang/assets/50468628/59aeed8e-db37-4c58-bab7-daaa35641d69">

Reply:
넵!